README.MD

```diff
@@ -57,6 +57,12 @@ O intuito deste repositório é mostrar projetos desenvolvidos para facilitar o
 | Laravel | Introdução ao Laravel para melhorar o conhecimento em PHP e sobre o padrão MVC | [Mateus Koppe](https://github.com/mateusKoppe) | <img alt="Badge em breve" src="https://img.shields.io/badge/-EM%20BREVE-red"> |
 | React | Aprenda o básico do React para construir aplicações Web de alta qualidade. | [Anna Campelo](https://github.com/AnnaCampelo) - [Lara Beatriz](https://github.com/larabeatrizms) |  <img alt="Badge em breve" src="https://img.shields.io/badge/-EM%20BREVE-red"> |-->
 
+### :triangular_ruler: Design & Architecture Patterns
+
+| Nome  | Descrição                                                                                                                                 | Contribuidores                                | Link                                                        |
+|-------|-------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|-------------------------------------------------------------|
+| SOLID | SOLID é um acrônimo para 5 regras que facilitam o desenvolvimento de  software para que ele seja flexível, legível e de fácil manutenção. | [Daniel Reis](https://github.com/DanielHe4rt) | [Clique aqui ➡️](https://github.com/DanielHe4rt/SOLID4Noobs) |
```

Comment:
Bom, achei bastante legal o propósito do repositório, porem tem alguns pontos que acho importante.
- Em algumas páginas seria legal comentar dentro do código para ficar mais claro para quem está lendo e não tem experiência com a linguagem. (Igual nesse primeiro [snippet](https://github.com/DanielHe4rt/SOLID4Noobs/blob/main/docs/1-srp.md#1---single-responsibility-principle))
Como aqui: [3-Isp.md](https://github.com/DanielHe4rt/SOLID4Noobs/blob/main/docs/3-lsp.md)
Entendo que é complicado explicar um princípio sem exemplos, mas seria legal comentários, até fiquei perdido nos snippet :joy:.
- Link de próxima página está bugado. [3-Isp.md](https://github.com/DanielHe4rt/SOLID4Noobs/blob/main/docs/3-lsp.md)

Pois o Solid pode ser usado em outros projetos e não só em PHP, creio que outras pessoas poderia ter problemas para entender o conceito que seria o mais importante independente da linguagem apresentada.

Reply:
Sobre o link, acabei de revisar e tudo certo.

O SOLID em si não sei se entraria como algo "4noobs" da vida pois o conceito em si é algo pra galera que já tenha experiência. Em tese, a linguagem não deveria impactar sobre os conceitos. 

Tudo ali é falado sobre responsabilidades e com exemplos de antes e depois do refactor, a legibidade deveria ser a mesma já que o principio de OO é universal.

Se for o caso, nós podemos planejar alguma série 4juniors e pensar sobre.

Reply:
Entendi, minha ideia ali era tentar facilitar para quem não tem domínio da syntax e queira entender o conceito.

Reply:
Acho que o que o @kjkGustavo quis dizer é que alguns pontos dos exemplos ficou confuso para entender o conceito em si.
 
Creio que adicionar alguns comentários nos snippets sobre o que ta acontecendo e como o SOLID está sendo aplicado naquele código.

Reply:
Vou tentar melhorar a explicação dos snippets e retorno o review.